azext_iot/digitaltwins/params.py

```diff
@@ -456,6 +456,14 @@ def load_digitaltwins_arguments(self, _):
             "In case of an error processing a batch, the behavior is determined by the --failure-policy parameter. ",
             arg_group="Models Input",
         )
+        context.argument(
```

Comment:
You are able to add a type constraint to the argument definition https://github.com/Azure/azure-iot-cli-extension/blob/dev/azext_iot/_params.py#L880